# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1310837627 14400
#      Sat Jul 16 13:33:47 2011 -0400
# Node ID e0b174e994492d4755042d27addc39bdc6e6f8a4
# Parent  6fdf4927fefc718333b169e0564665f2264de3f3
codesprint: fix comment

* wavread.m: Use same style of comment for assert(1) test that
is used in other files.

diff --git a/scripts/audio/wavread.m b/scripts/audio/wavread.m
--- a/scripts/audio/wavread.m
+++ b/scripts/audio/wavread.m
@@ -235,12 +235,11 @@ function chunk_size = find_chunk (fid, c
     chunk_size += rem (chunk_size, 2);
     offset = offset + 8 + chunk_size;
   endwhile
   if (! strcmp (id, chunk_id))
     chunk_size = -1;
   endif
 endfunction
 
-
-%% Tests for wavread/wavwrite pair are in wavrite.m
-%!assert(1)  # stop fntests.m from reporting no tests for wavread
-
+## Mark file as being tested.  Tests for wavread/wavwrite pair are in
+## wavrite.m
+%!assert(1)
